src/app/meme-dialog/meme-dialog.component.html

```diff
@@ -15,7 +13,25 @@
       <button mat-icon-button (click)="onClickDownVote()" [color]="myVote == -1 ? 'warn': 'basic'"><i
         class="material-icons">thumb_down</i></button>
     </div>
-
+    <mat-list>
+      <mat-divider></mat-divider>
+      <mat-list-item *ngFor="let comment of comments">
+        <div class="comment-line">
+          <div class="comment">
+            <div class="comment-info" matLine>{{ comment.User.username }} commented {{ comment.createdAt | timeAgo }}</div>
+            <div matLine>{{ comment.text }}</div>
+          </div>
+          <div class="delete-button" *ngIf="userService.user && comment.User.username===userService.user.username">
+            <button mat-icon-button (click)="deleteComment(comment.id)"><i class="material-icons">delete</i></button>
+          </div>
+        </div>
+        <mat-divider [inset]="true" ></mat-divider>
+      </mat-list-item>
+    </mat-list>
+    <form [formGroup]="form" (ngSubmit)="submitComment()" *ngIf="userService.isLoggedIn()">
+      <mat-form-field>
+        <input matInput type="text" placeholder="Add a comment" formControlName="commentText" maxLength="1000">
```

Comment:
I think this should be a textarea

Reply:
Can you bind `(onEnter)`?

Reply:
Also doesn't work :/

Reply:
That makes sense, since you press enter to make new lines in the text area.

You probably want to go with a separate "Post" button.

Reply:
Sure; should I use the send icon or the comment icon do you think? Also @xiningchen 
https://material.io/icons/#ic_send
https://material.io/icons/#ic_comment